pull-kubernetes-cross is failing

<!-- This form is for bug reports and feature requests ONLY! 

If you're looking for help check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
-->

**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
> /kind feature


**What happened**: `pull-kubernetes-cross` is failing with:

```
I1118 20:53:49.950] cmd/kubelet/app/server.go:417:49: too many arguments in call to cadvisor.New
I1118 20:53:49.950] 	have (string, uint, cadvisor.ImageFsInfoProvider, string, bool)
I1118 20:53:49.950] 	want (string, uint, cadvisor.ImageFsInfoProvider, string)
```

**What you expected to happen**: The cross build should work,.

**How to reproduce it (as minimally and precisely as possible)**: `/test pull-kubernetes-cross`


**Anything else we need to know?**:

It looks like this happened in https://github.com/kubernetes/kubernetes/pull/55642#issuecomment-345462139, probably because we don't block the queue on cross :-(
xref: https://github.com/kubernetes/test-infra/issues/5584

/priority failing-test
/sig release

--- a/pkg/kubelet/cadvisor/cadvisor_unsupported.go
+++ b/pkg/kubelet/cadvisor/cadvisor_unsupported.go
@@ -31,7 +31,7 @@ type cadvisorUnsupported struct {
 
 var _ Interface = new(cadvisorUnsupported)
 
-func New(address string, port uint, imageFsInfoProvider ImageFsInfoProvider, rootPath string) (Interface, error) {
+func New(address string, port uint, imageFsInfoProvider ImageFsInfoProvider, rootPath string, usingLegacyStats bool) (Interface, error) {
 	return &cadvisorUnsupported{}, nil
 }
 

--- a/pkg/kubelet/cadvisor/cadvisor_windows.go
+++ b/pkg/kubelet/cadvisor/cadvisor_windows.go
@@ -32,7 +32,7 @@ type cadvisorClient struct {
 var _ Interface = new(cadvisorClient)
 
 // New creates a cAdvisor and exports its API on the specified port if port > 0.
-func New(address string, port uint, imageFsInfoProvider ImageFsInfoProvider, rootPath string) (Interface, error) {
+func New(address string, port uint, imageFsInfoProvider ImageFsInfoProvider, rootPath string, usingLegacyStats bool) (Interface, error) {
 	client, err := winstats.NewPerfCounterClient()
 	return &cadvisorClient{winStatsClient: client}, err
 }

